sklearn/multioutput.py

```diff
@@ -1,26 +1,32 @@
 """
-This module implements multioutput regression.
+This module implements multioutput regression and classification.
 
 The estimator provided in this module is a meta-estimator: it requires
```

Comment:
Had to rebase on master. Forgot this grammatical change. Thanks @betatim for having look at it. Will reread it and check for any remaining changes. Would be glad to incorporate if you have any other inputs.